platform/saivpp/vpplib/vppxlate/SaiVppXlate.h

```diff
@@ -81,6 +81,21 @@ extern "C" {
         vpp_acl_rule_t rules[0];
     } vpp_acl_t;
 
+    typedef struct {
+        vpp_ip_addr_t dst_prefix;
+        vpp_ip_addr_t dst_prefix_mask;
+        char hwif_name[64];
+        uint8_t  ip_protocol;
+        vpp_ip_addr_t next_hop_ip;
+    } tunterm_acl_rule_t;
+
+    typedef struct _vpp_tunterm_acl_ {
+        char *acl_name;
+        uint32_t count;
+        tunterm_acl_rule_t rules[0];
+    } vpp_tunerm_acl_t;
```

Comment:
Correct structure name to vpp_tunterm_acl_t

Reply:
Updated